ReviewMonitor/AppDelegate.swift

```diff
@@ -84,7 +84,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         addBlurView()
     }
 
+    func applicationDidBecomeActive(_ application: UIApplication) {
+        ReachabilityManager.shared.startMonitoring()
```

Comment:
Why do we need stop and start monitoring when app good es to background and comes to foreground? Why can't we leave at the monitoring state even when the app goes to background?